astropy/units/core.py

```diff
@@ -113,14 +114,25 @@ class _UnitRegistry(object):
     Manages a registry of the enabled units.
     """
     def __init__(self, init=[], equivalencies=[]):
-        if isinstance(init, _UnitRegistry):
-            equivalencies = init.equivalencies
-            init = init.all_units
-
         self._reset_units()
```

Comment:
I thought about a `copy` method but decided against it because it's a private class (leading underscore) and only `add/set_enabled_units/equivalencies` do use it.

I got sidetracked by trying to implement it as `ChainMap` (which would also make all the copies unnecessary) until I realized that it's not avaiable in python2. :smile: Need to think about it some more. :smile: